# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/dialog/menubox-4-2.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./menubox.c.orig	2003-03-08 18:40:50.000000000 +0200
+++ ./menubox.c	2003-06-28 19:33:18.000000000 +0300
@@ -77,14 +77,14 @@
     /* Draw the input field box (only for inputmenu) */
     (void) wmove(win, y + 1, item_x);
     if (dialog_vars.input_menu)
-	draw_box(win, y, item_x, INPUT_ROWS, menu_width - 2 - item_x - tag_x,
+	draw_box(win, y, item_x, INPUT_ROWS, menu_width - item_x - tag_x,
 		 selected ? item_selected_attr : item_attr,
 		 selected ? item_selected_attr : item_attr);
 
     /* print actual item */
     wmove(win, y + 1, item_x + 1);	/* go into inputbox */
     wattrset(win, selected ? item_selected_attr : item_attr);
-    dlg_print_text(win, ItemText(0), menu_width - 2 - item_x - 2, &attr);
+    dlg_print_text(win, ItemText(0), menu_width - item_x - 2, &attr);
 
     if (selected) {
 	dlg_item_help(ItemHelp(0));
@@ -132,7 +132,7 @@
 	     *                      ( see in dialog_menu() )
 	     */
 	    dlg_show_string(win, result, offset, item_selected_attr,
-			    y + 1, item_x + 1, menu_width - 2 - item_x - 2,
+			    y + 1, item_x + 1, menu_width - item_x - 2,
 			    FALSE, first);
 	    first = FALSE;
 	}
